tests/ops/functions/test_map_wires.py

```diff
@@ -91,7 +91,7 @@ def test_map_wires_with_queuing_and_with_replacing(self):
 
     def test_map_wires_unsupported_object_raises_error(self):
         """Test that an error is raised when trying to map the wires of an unsupported object."""
-        with pytest.raises(ValueError, match="Cannot map wires of object"):
+        with pytest.raises(qml.transforms.core.TransformError, match="Decorating a QNode with"):
```

Comment:
Wait, why does the error talk about QNodes? Is this an informative error?

Reply:
This kind of syntax is what was used for something like:
```
@qml.map_wires({0:1, 1:2})
@qml.qnode(dev)
...
```

That is now removed.  Potentially we could update that error message now though.